sys-cluster/lmod/lmod-8.4.20.ebuild

```diff
@@ -0,0 +1,137 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1..3} )
+inherit autotools lua-single
+
+DESCRIPTION="Environment Module System based on Lua"
+HOMEPAGE="https://lmod.readthedocs.io/en/latest"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/TACC/Lmod"
+else
+	SRC_URI="https://github.com/TACC/Lmod/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}"/Lmod-${PV}
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="auto-swap +cache +dotfiles duplicate +extend italic module-cmd +nocase redirect test"
+REQUIRED_USE="${LUA_REQUIRED_USE}"
+RESTRICT="!test? ( test )"
+
+RDEPEND="${LUA_DEPS}
+	app-shells/tcsh
+	|| (
+		app-shells/loksh
+		app-shells/mksh
+		app-shells/ksh
+	)
+	app-shells/zsh
```

Comment:
Why this weird way of requiring shell? At least ctrl+f'ing configure.ac doesn't have tcsh listed.

Reply:
see below.